cli/src/format.rs

```diff
@@ -172,7 +172,7 @@ impl Format for Package {
             ["Author Risk:", r -> risk_to_string(self.risk_scores.author)],
             ["Engineering Risk:", r -> risk_to_string(self.risk_scores.engineering)],
             ["License Risk:", r -> risk_to_string(self.risk_scores.license)],
-            ["Malicious Code Risk:", r -> risk_to_string(self.risk_scores.malicious_code)],
+            ["Malicious Risk:", r -> risk_to_string(self.risk_scores.malicious)],
```

Comment:
```suggestion
            ["Malicious Code Risk:", r -> risk_to_string(self.risk_scores.malicious)],
```
> The breaking changes also introduce changes to the CLI's output. I've made these changes for the sake of consistency, but if we still want to advertise this as "malicious code" to our users we can keep the output unchanged.

Just an opinion...but keeping the output as `Malicious Code Risk` just reads a bit better...slightly more clear. The opinion is not strong enough to hold this up...keep it if preferred.

Reply:
I kinda agree, but I'm not sure if there's a deeper meaning behind removing the "code" part.

Reply:
The change was made in https://github.com/phylum-dev/phylum-types/pull/66 ...but there is not a corresponding issue to explain the rationale behind the change further. I _think_ I saw some traffic on this in chat about how there are multiple names for this same risk type and _something_ needed to be selected to rally around. IIRC `malicious` was chosen b/c it was the shortest. I don't believe the "text" output form of this risk domain has to match the "code" reference name verbatim, though.

CC: @louislang @matt-phylum (for any additional context)